### ISSUE TITLE
Some questions on the sceneflow dataset

### ISSUE THREAD

**Issue by Jasookii** [songlin/d3roma#20]
First of all, thank you for the authors' excellent work. I have the following questions:

The article mentions the training and validation results obtained solely on the SceneFlow dataset. Could you release the corresponding pre-trained weights that are trained only on the SceneFlow dataset?
It was mentioned that libSGM was used to precompute disparity maps for SceneFlow. What is the difference between the disparity maps generated in this step and the disparity GT that comes with the SceneFlow dataset?
It was mentioned that some challenging stereo images were removed during the training process. Are the validation results for SceneFlow in the article also the ones after these stereo images have been removed?

**Comment by songlin** [comment]
Hi Jasookii,

Thanks for paying attention to our work.

I am afraid that the weights for sceneflow are not going to be released because I did not save them. However, you can follow the script to train yourself. We use libSGM to compute the disparity maps to mimic camera sensor disparity characteristics. As you noticed, we remove the stereo images which are too challenging due to large occusion or severe motion blur from training set.  We did not  filter any data from validation set.  However, we report the results at resolution 420x270 and caping the maximum disparity at 192 if i remember correctly.

**Comment by Jasookii** [comment]
Thanks，is the training process supervised using the disparity maps generated by libSGM?

**Comment by songlin** [comment]
No, the noisy disparity generate by libSGM is only used as condition to diffusion models, the model then learns to recover them and thus is supervised by the ground truth disparity.

**Comment by Jasookii** [comment]
Thank you, author, for your patient reply. 

I would like to ask if you have tried the solution of freezing the weights of stable diffusion (more regular and cheap way), and is the effect different from fine-tuning stable diffusion?

**Comment by songlin** [comment]
You are welcome!

I am not sure I understand your question. We finetune stable diffusion 2.1 which is a text-to-image generation model and adapt it to an image-to-image translation model. Hence, the model cannot be used as it were. Maybe you are talking about some other researches that tried to use a freezed diffusion model as the feature extractor and train a custom depth prection module?

**Comment by Jasookii** [comment]
I list some solutions of freezing the weights of stable diffusion and adding additional control information:
Adding Conditional Control to Text-to-Image Diffusion Models
Exploiting Diffusion Prior for Real-World Image Super-Resolution
DiffBIR: Towards Blind Image Restoration with Generative Diffusion Prior

**Comment by Jasookii** [comment]
Hello, I would like to ask how much training time is under 8*A100 40G

**Comment by songlin** [comment]
Hi Jasookii,

Indeed, those papers you mentioned are alternative ways to capitalize on the pretrained knowledge from stable diffusions. However, we found that full finetuning also works well by simply adapting the first input layer. 

Our training time on mixed data of SceneFlow, DREDS, and HISS takes about 10 hours on 8*A100 40G.

Best,

**Comment by Jasookii** [comment]
When using SceneFlow for training and testing, is it true that training uses finalpass + cleanpass, while testing only uses cleanpass? Have you ever tested the results on finalpass?
